https/formData.go

```diff
@@ -207,6 +203,21 @@ func (fp *formParam) processSlice() (map[string][]string, error) {
 	return result, nil
 }
 
+func initIndexString(elem reflect.Value, index int) any {
```

Comment:
this looks like unreadable code. can you please refactor this one?

Reply:
Done refactoring to make it readable.